.github/workflows/publish-packages.yml

```diff
@@ -91,7 +91,9 @@ jobs:
           ./build-hazelcast-deb-package.sh
 
       - name: Create & Sign & Upload RPM package
+        uses: docker://rockylinux:latest
```

Comment:
nice, didn't know about it

Reply:
It doesn't work exactly as I wanted - you can't use both `uses` and `run` at the same time - this basically invokes an action specified in the container.

Need to rework this.